letsencrypt/storage.py

```diff
@@ -157,16 +165,24 @@ def _consistent(self):
                 self.cli_config.archive_dir, self.lineagename)
             if not os.path.samefile(os.path.dirname(target),
                                     desired_directory):
+                logger.debug("Element's link does not point within the "
+                             "cert lineage's directory within the "
+                             "official archive directory. Link: %s, "
+                             "archive directory: %s.",
+                             os.path.dirname(target), desired_directory)
                 return False
 
             # The link must point to a file that exists
             if not os.path.exists(target):
+                logger.debug("Link %s points to a file that does not exist.", target)
                 return False
 
             # The link must point to a file that follows the archive
             # naming convention
             pattern = re.compile(r"^{0}([0-9]+)\.pem$".format(kind))
             if not pattern.match(os.path.basename(target)):
+                logger.debug("Link %s does not follow the archive naming "
+                             "convention.", os.path.basename(target))
```

Comment:
This should probably be `logger.debug("%s does not follow the archive naming convention.", target)`.